frontend/micro-ui/web/micro-ui-internals/packages/Mukta/src/components/WorksCard.js

```diff
@@ -4,7 +4,7 @@ import { useTranslation } from "react-i18next";
 
 const ROLES = {
   PROJECT: ["PROJECT_CREATOR", "PROJECT_VIEWER"],
-  ESTIMATE: ["ESTIMATE_CREATOR", "ESTIMATE_VERIFIER", "TECHNICAL_SANCTIONER", "ESTIMATE_APPROVER"],
+  ESTIMATE: ["ESTIMATE_CREATOR", "ESTIMATE_VERIFIER", "TECHNICAL_SANCTIONER", "ESTIMATE_APPROVER","ESTIMATE_VIEWER"],
```

Comment:
@nipunarora-eGov  just check once, viewer we dont have to give estimate inbox screen , let him get only search screen access through side bar and when he click on create contract he navigates to search estimate